Readme typo fixes (#1319)
api3dao/airnode: packages/airnode-deployer/README.md

# Edit both `config.json` and `secrets.env` to reflect your configuration
```

> After you prepare the necessary deployment files, there are two options how to run the CLI:

Reviewer: ```suggestion
After you prepare the necessary deployment files, there are two options for running the CLI:
```
